ilc/client/CanonicalTagHandler.ts

```diff
@@ -0,0 +1,63 @@
+import { IlcIntl } from 'ilc-sdk/app';
+import singleSpaEvents from './constants/singleSpaEvents';
+import { removeQueryParams } from '../common/utils';
+import { Route } from '../server/types/RegistryConfig';
+import type { Logger } from 'ilc-plugins-sdk';
+
+interface ClientRouter {
+    getCurrentRoute(): Route;
+}
+
+export class CanonicalTagHandler {
```

Comment:
So, basically we are changing the href to canonicals if I'm not misreading this. Could u please explain where this requirement is coming from, my though was we need to update the tag itself